### PR TITLE
Feature/rtrail 05

### PR DESCRIPTION
**Pull request by DNizhebortsev**

### CODE REVIEW

File: src/main/java/io/khasang/rtrail/dao/impl/LocationDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import io.khasang.rtrail.dao.LocationDao;
+
+import io.khasang.rtrail.entity.Location;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+public class LocationDaoImpl extends BasicDaoImpl<Location> implements LocationDao{
+    @Autowired
+    private LocationDao locationDao;
```
Comment (SKuznet) [inline src/main/java/io/khasang/rtrail/dao/impl/LocationDaoImpl.java]: need empty space here

File: src/test/java/io/khasang/rtrail/controller/LocationControllerIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    private final static String ROOT = "http://localhost:8080/location";
+    private final static String ADD = "/add";
+    private final static String UPDATE = "/update";
+    private final static String DELETE = "/delete";
+    private final static String ALL = "/all";
+    private final static String GET_BY_ID = "/get";
+
+    @Before
+    public void init() {
+
```
Comment (SKuznet) [inline src/test/java/io/khasang/rtrail/controller/LocationControllerIntegrationTest.java]: why it here?

File: src/test/java/io/khasang/rtrail/controller/LocationControllerIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+                null,
+                new ParameterizedTypeReference<List<Location>>() {
+                }
+        );
+
+        List<Location> locations = responseEntity.getBody();
+        assertNotNull(locations.get(0));
+        assertNotNull(locations.get(1));
+
+        // clean
```
Comment (SKuznet) [inline src/test/java/io/khasang/rtrail/controller/LocationControllerIntegrationTest.java]: -

File: src/test/java/io/khasang/rtrail/controller/LocationControllerIntegrationTest.java [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+        assertEquals("OK", responseEntity.getStatusCode().getReasonPhrase());
+
+        Location recievedLocation = responseEntity.getBody();
+        assertNotNull(recievedLocation);
+    }
+
+    @Test
+    public void checkAllLocations() {
+        // clean
```
Comment (SKuznet) [inline src/test/java/io/khasang/rtrail/controller/LocationControllerIntegrationTest.java]: -

**Review by SKuznet — DISMISSED** [review]

**Comment by DNizhebortsev** [comment]
corrected, please check

**Review by SKuznet — APPROVED** [review]

**Pull request merged**